### ISSUE TITLE
More granular github permissions

### ISSUE THREAD

**Issue by SusanBrown** [cwrc/CWRC-GitWriter#25]
Does the authorization you have to provide to use gitWriter give CWRC as an org the ability to write to **any** of the user's github accounts, or just those that are created by/shared with Git-writer?

**Comment by jchartrand** [comment]
Any.  

From the Github blog (https://blog.github.com/2012-09-21-easier-builds-and-deployments-using-git-over-https-and-oauth/):

"While Git over HTTP with OAuth reduces friction for some types of applications, keep in mind that unlike deploy keys, OAuth tokens work for any repository for which the user has access."

An alternative might be for someone to create another Github account for their CWRC work.

**Comment by SusanBrown** [comment]
Thanks, James. 

Mihaela, could you please add this to the troubleshooting section of the documentation that you are starting? (perhaps creating a subsection for GitWriter?)

**Comment by ilovan** [comment]
Yes, I will do it, though I think not to troubleshooting, since it's not about troubleshooting per se. I will add it as a Warning message in the DITA document on how to open CWRC Writer (when we mention giving Git_writer permission to access the GitHub account).
